libraries/Microsoft.Bot.Builder.Dialogs/Prompts/PromptOptions.cs

```diff
@@ -51,5 +51,12 @@ public class PromptOptions
         /// </summary>
         /// <value>Additional options for use with a prompt validator.</value>
         public object Validations { get; set; }
+
+        /// <summary>
+        /// Gets or sets additional options passed to the underlying
+        /// <see cref="ChoiceRecognizers.RecognizeChoices(string, IList{Choice}, FindChoicesOptions)"/> method.
+        /// </summary>
+        /// <value>Options to control the recognition strategy.</value>
+        public FindChoicesOptions RecognizerOptions { get; set; }

```

Comment:
Hi @rodolZ80, we are still reviewing this PR and see if we can find a different way of supporting your scenario. 
The main issue here is that we are adding a `FindChoicesOptions` property to `PromptOptions` that is only meaningful to `ChoicePrompt` and it is not relevant in all the other classes derived from `Prompt<T>`.
If you are blocked by this, the only short term solution I can think off right now is to create your own specialized `ChoicePrompt` and `PromptOptions` that override `OnRecognizeAsync()` and set the expected `FindChoicesOptions` 
Will post back here if we can find a better solution